docs/contracts/abstract-contracts.rst

```diff
@@ -6,27 +6,31 @@
 Abstract Contracts
 ******************
 
-Contracts are marked as abstract when at least one of their functions lacks an implementation as in the following example (note that the function declaration header is terminated by ``;``)::
+Contracts need to be marked as abstract when at least one of their functions was not implemented.
+This can be done by using the ``abstract`` keyword as shown in the following example. Note, that this contract need to be
+defined as abstract, because the function ``utterance()`` was defined, but no implementation was provided (no implementation body ``{ }`` was given).::
```

Comment:
Please put the `::` on a line of its own.

Reply:
It looks like that it is not possible to just have `::` in one line.